packages/react-charting/src/components/HeatMapChart/HeatMapChart.base.tsx

```diff
@@ -356,28 +356,28 @@ export class HeatMapChartBase extends React.Component<IHeatMapChartProps, IHeatM
    * and un-highlight the rest of them
    * @param legendTitle
    */
-  private _onLegendHover = (legendTitle: string): void => {
+  private _onLegendHover(legendTitle: string): void {
```

Comment:
@srmukher Is this change still needed to run unit tests.

Reply:
Yes @AtishayMsft, this is required as "jest.spyOn(HeatMapChartBase.prototype as any, '_onLegendHover');" requires the function to be a classic non-arrow function